otter/test/convergence/test_service.py

```diff
@@ -104,24 +108,33 @@ def test_buckets_acquired(self):
         When buckets are allocated, the result of converge_all_groups is
         performed.
         """
-        def converge_all_groups(log, currently_converging, _my_buckets,
+        def converge_all_groups(currently_converging, _my_buckets,
                                 all_buckets, divergent_flags):
             return Effect(
-                ('converge-all', log, currently_converging, _my_buckets,
+                ('converge-all', currently_converging, _my_buckets,
                  all_buckets, divergent_flags))
 
+        def perform_ca_eff(bound_fields):
```

Comment:
I would prefer to move @cyli's `_unwrap_wrapped_effect` from `test_controller.py` to `test.utils` and use it here. I think it allows the code to read a bit better, since you can nest the intents within the main sequence, showing the linear flow of effects mapped directly to linear code